### DISCUSSION TITLE
QuestionAnsweringTrainer for adapter?

### DISCUSSION THREAD

**Discussion by keyuchen21** [adapter-hub/adapters#689] [Q&A] [answered]
in huggingface's examples, I saw the QuestionAnsweringTrainer modual, are there similar trainer in adapters?
if no, are there any example for train QA tasks?

https://github.com/huggingface/transformers/blob/main/examples/pytorch/question-answering/run_qa.py
<img width="880" alt="image" src="https://github.com/adapter-hub/adapters/assets/54015474/e79a488b-980a-4b06-a096-8256d7cbc1a3">

**Comment by lenglaender** [comment, marked as answer]
Hi @km5ar,

the _Adapters_ library provides adapted versions of many Hugging Face scripts in the `examples/` directory, including the adapted question answering scripts. You can find the adapted script and `QuestionAnsweringTrainer` here: https://github.com/adapter-hub/adapters/tree/main/examples/pytorch/question-answering
